Apps/W1/EDocumentConnectors/Continia/app/src/Onboarding/ContiniaParticipation.Table.al

```diff
@@ -0,0 +1,290 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector.Continia;
+
+using System.Email;
+using Microsoft.Finance.VAT.Registration;
+using Microsoft.Foundation.Address;
+
+table 6391 "Continia Participation"
```

Comment:
We have a table called Service Participation in E-Doc Core. 
Maybe we can build of on that? Or if it does not make sense, then that's also okay.